varia/sass/vendors/_jetpack.scss

```diff
@@ -34,3 +34,75 @@
 		margin-bottom: 0;
 	}
 }
+
+/**
+ * Cookies & Consents Banner
+ */
+body {
```

Comment:
Note: We need to use `body` to make sure we actually overwrite Jetpack's styles.